quickstarts/host-inventory-quickstart.yaml

```
@@ -12,6 +12,7 @@ spec:
   durationMinutes: 20
   description: Configure your host inventory and create a cluster.
   prerequisites: 
+    - 'You completed the "Get started with multicluster engine"'
```

Just a suggestion for clarity:
```suggestion
    - 'You have completed the steps to "Get started with multicluster engine"'
```